main_2024.py

```diff
@@ -357,24 +395,14 @@ def main() -> int:
                 return -1
 
         try:
-            geolocation_data = geolocation_to_main_queue.queue.get_nowait()
+            cluster_estimation_data = cluster_estimation_to_main_queue.queue.get_nowait()
         except queue.Empty:
-            geolocation_data = None
-
-        if geolocation_data is not None:
-            for detection_world in geolocation_data:
-                main_logger.debug("Detection in world:", True)
-                main_logger.debug(
-                    "geolocation vertices: " + str(detection_world.vertices.tolist()), True
-                )
-                main_logger.debug(
-                    "geolocation centre: " + str(detection_world.centre.tolist()), True
-                )
-                main_logger.debug("geolocation label: " + str(detection_world.label), True)
-                main_logger.debug(
-                    "geolocation confidence: " + str(detection_world.confidence), True
-                )
+            cluster_estimation_data = None
 
+        if cluster_estimation_data is not None:
+            for object_in_world in cluster_estimation_data:
+                main_logger.debug("Cluster in world: ", (object_in_world is not None))
+                main_logger.debug(str(object_in_world))
```

Comment:
While its good to check `object_in_world` is not `None`, I don't think its needed given that in the `ClusterEstimation.run()` it already checks that there are no `None` values in the list. Replace `(object_in_world is not None)` with `True`.